pkgs/top-level/all-packages.nix

```diff
@@ -39673,7 +39673,10 @@ with pkgs;
 
   appcsxcad = libsForQt5.callPackage ../applications/science/electronics/appcsxcad { };
 
-  simulide = libsForQt5.callPackage ../applications/science/electronics/simulide { };
+  inherit (libsForQt5.callPackage ../applications/science/electronics/simulide { })
```

Comment:
Ideally we should use callPackages (notices the s) here for override to properly work, but I don't know how to make it work with libsForQt5.

Reply:
Maybe I could separate the versions into different small `.nix` files, and call them one-by-one by path in `all-packages.nix`. Though I am not sure if that would solve the issue completely, because the generic builder currently uses packages from the same place for all versions.